spec/tmp/.gitignore

```diff
@@ -0,0 +1 @@
+*
```

Comment:
Can this be a part of the .gitignore in the root?

Reply:
The file serves two purposed. It ensures the repository has a `tmp` folder for the specs and it ensures there is never any files in the folder. I could be in the root .gitignore, but then there would be some kind of `.keep` to ensure the foder would be in git.